dgraph/cmd/zero/http.go

```diff
@@ -231,9 +235,60 @@ func (st *state) getState(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) zeroHealth(ctx context.Context, all bool) (*api.Response, error) {
+	if ctx.Err() != nil {
+		return nil, ctx.Err()
+	}
+
+	var healthAll []pb.HealthInfo
+
+	// Append self.
+	healthAll = append(healthAll, pb.HealthInfo{
+		Instance:    "zero",
+		Address:     x.WorkerConfig.MyAddr,
+		Status:      "healthy",
+		Group:       strconv.Itoa(int(worker.GroupId())),
+		Version:     x.Version(),
+		Uptime:      int64(time.Since(x.WorkerConfig.StartTime) / time.Second),
+		LastEcho:    time.Now().Unix(),
+	})
+
+	var err error
```

Comment:
declaration is probably not needed. And can move out the json.Marshal line out of the if